### DISCUSSION TITLE
no USB com port on UGS platform v2.0.7 to latest build (macOS)

### DISCUSSION THREAD

**Discussion by lrovelli82** [winder/Universal-G-Code-Sender#1694] [Help]
Cannot see any usb com ports on UGS platform starting from v2.0.7 onwards on my MBA running High Sierra (10.13.6). I am using the "all platform" option and UGS platform runs off the same folder on Windows (both 32bit and 64bit) without any issue with my CNC 3080pro. Both macOS and Windows have JRE 8u251. 
I tried swapping the jssc.jar file (as some have suggested) but I still cannot see any com port (only bluetooth and wireless). Also tried the latest JRE update with no luck. Any known workarounds or solution would be very appreciated ;-)

**Comment by breiler** [comment]
Difficult to answer without knowing what hardware you are using but this is commonly a driver problem. Have a look here and see if anything of this helps: https://github.com/winder/Universal-G-Code-Sender/wiki/Connecting-the-Controller
